src/lib/core/styles/tokens/avatar/_tokens.scss

```diff
@@ -0,0 +1,24 @@
+@use 'sass:map';
+@use '../../animation';
+@use '../../border';
+@use '../../elevation';
+@use '../../shape';
+@use '../../spacing';
+@use '../../theme';
+@use '../../typography';
```

Comment:
Let's remove any unused modules from here. Not critical, but trying to keep these clean so we can quickly see what global tokens are inherited.